MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -344,7 +344,18 @@ open class RouteLegProgress: NSObject {
         let upcomingCoords = upComingStep?.coordinates ?? []
         let currentCoords = currentStep.coordinates ?? []
         let nearby = priorCoords + currentCoords + upcomingCoords
-        assert(!nearby.isEmpty, "Polyline must coordinates")
+        assert(!nearby.isEmpty, "Step must have coordinates")
+        return nearby
+    }
+    
+    /**
+     Returns an array of `CLLocationCoordinate2D` of the current and upcoming step geometry.
+     */
+    public var currentAndUpcomgingStepCoordinates: [CLLocationCoordinate2D] {
+        let upcomingCoords = upComingStep?.coordinates ?? []
+        let currentCoords = currentStep.coordinates ?? []
+        let nearby = currentCoords + upcomingCoords
```

Comment:
Is a separate method really needed to concatenate these two arrays, given that there’s only one call site?